### ISSUE TITLE
zod .optional() does not work in v0.8.0-pre.0

### ISSUE THREAD

**Issue by fiws** [edmundhung/conform#235]
We've been testing [v0.8.0-pre.0](https://github.com/edmundhung/conform/commit/8106af248fed9278b021100263ac6a5f2bc2d0ee) and noticed that `.optional()` does nothing in that version. It seems like there is no way to mark a field as optional, we get "Required" as error regardless of what we do.

Example schema:
```
const schema = z.object({
  email: z
    .string()
    .optional().nullable()
});
```

Sending an empty string always returns "Required" as error.

**Comment by edmundhung** [comment]
Good catch, @fiws! That's indeed a problem I overlooked. Working on a solution now.

**Comment by edmundhung** [comment]
This is fixed on `v0.8.0-pre.1`. Thanks again!
